flow360/component/flow360_params/flow360_params.py

```diff
@@ -155,36 +155,42 @@ class MeshBoundary(Flow360BaseModel):
 class Boundary(Flow360BaseModel, metaclass=ABCMeta):
     """Basic Boundary class"""
 
-    type: str
+    model_type: str
```

Comment:
why change `type` to `model_type` here? `type` is required property by solver

Reply:
Reverted to "type" for boundaries